src/pkgs/files/login-inner.nix

```diff
@@ -1,14 +1,14 @@
 # Licensed under GNU Lesser General Public License v3 or later, see COPYING.
 # Copyright (c) 2019 Alexander Sosedkin and other contributors, see AUTHORS.
 
-{ initialBuild, instDir, packageInfo, writeText }:
+{ initialBuild, instDir, packageInfo, writeTextDir }:
 
-writeText "login-inner" ''
+writeTextDir "bin/login-inner" ''
   set -e
 
   [ "$#" -gt 1 ] || echo "Welcome to Nix-on-Droid!"
 
-  [ "$#" -gt 1 ] || echo "If nothing works, use the rescue shell and read ${instDir}/bin/.login-inner"
+  [ "$#" -gt 1 ] || echo "If nothing works, use the rescue shell and read ${instDir}/bin/login-inner"
```

Comment:
Why do you rename `.login-inner`? The previous rationale for hiding it was that I couldn't come up with a scenario where it wasn't used without `login`.

Reply:
The main reason was to silence `writeTextDir` because it generates the derivation name from the file name. But derivation names must not start with a dot..  Furthermore `login-inner` is no executable anymore.

To be honest, it was a quick fix, but I don't know if `login-inner` really belongs into `/bin`.

Reply:
TBH, it does not. You can put it somewhere else, sounds right.

Reply:
Do have a fitting path in mind?

Reply:
Not really. I'd probably go with something like `.../usr/helpers/.login-inner`, but I have no strong preference.